routes.py

```
@@ -184,63 +241,32 @@ def get_jellyfin_metadata() -> ResponseReturnValue:
+                    extra_params={"PersonTypes": "Actor"},
+                ),
+                "tag": pool.submit(_fetch_jellyfin_endpoint, url, api_key, "Tags"),
             }
-        )
+            for key, future in futures.items():
+                try:
+                    items = future.result()
+                    result[key] = [
+                        item.get("Name", "") for item in items if item.get("Name")
+                    ]
+                except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)